topics/basics/testing_plugins/integration_tests/integration_tests_intro.md

```diff
@@ -0,0 +1,262 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# Creating the first integration test
+
+## Why create integration tests?
```

Comment:
Please replace this title with a simple introduction sentence that the list below completes.